x-pack/plugins/security_solution/server/lib/risk_engine/risk_weights.ts

```diff
@@ -28,7 +28,7 @@ const DEFAULT_CATEGORY_WEIGHTS: RiskWeights = RISK_CATEGORIES.map((category) =>
  * This function and its use can be deleted once we've replaced our use of event.kind with a proper risk category field.
  */
 const convertCategoryToEventKindValue = (category?: string): string | undefined =>
-  category === 'alerts' ? 'signal' : category;
+  category === 'category_1' ? 'signal' : category;
```

Comment:
Why do we need to rename alerts to `cateogry_1`

Reply:
This is discussed in the [ongoing ECS RFC(s)[https://github.com/elastic/ecs/pull/2232/files#diff-40e16fa32db2ffbdeabf56eb2ffeac75dfa12410e662c3328c7c859b426805d2R20], but the summary is: futureproofing.